api/default_networks.go

```diff
@@ -346,6 +347,39 @@ func baseSepolia(proxyHost, stageName string) params.Network {
 	}
 }
 
+func statusNetworkSepolia(proxyHost string) params.Network {
+	const chainID = common.StatusNetworkSepoliaChainID
+	const chainName = "status"
+	const networkName = "sepolia"
+
+	rpcProviders := []params.RpcProvider{
+		// Smart proxy provider
+		*params.NewEthRpcProxyProvider(chainID, StatusSmartProxy, smartProxyUrl(proxyHost, chainName, networkName), false),
+		// Direct providers
+		*params.NewDirectProvider(chainID, DirectStatus, "https://public.sepolia.rpc.status.network", false),
+	}
+
+	return params.Network{
+		ChainID:                chainID,
+		ChainName:              "Status Network Sepolia",
+		RpcProviders:           rpcProviders,
+		BlockExplorerURL:       "https://sepoliascan.status.network/",
+		IconURL:                "network/Network=Status-test",
+		ChainColor:             "#7140FD",
+		ShortName:              "status",
+		NativeCurrencyName:     "Ether",
+		NativeCurrencySymbol:   "ETH",
+		NativeCurrencyDecimals: 18,
+		IsTest:                 true,
+		Layer:                  2,
+		Enabled:                false,
```

Comment:
Why make it disabled by default?
With the current configuration the network is `Active` by default:
![image](https://github.com/user-attachments/assets/7b658200-eec6-42a3-b8cc-6932700cdeff)
But balances won't show up for this chain since it's not `Enabled`
![image](https://github.com/user-attachments/assets/6d586498-e128-4633-bac1-523b24254445)


We might as well make it enabled.

Reply:
All the Sepolia chains in this list (or file) were disabled by default. I followed the same for Status Network Sepolia. I will update it to be enabled by default 👍

Reply:
> All the Sepolia chains in this list (or file) were disabled by default. I followed the same for Status Network Sepolia. I will update it to be enabled by default 👍

ugh I hadn't noticed this. Actually ETH sepolia is enabled by default:
![image](https://github.com/user-attachments/assets/a04aa94e-9c10-47cc-8cf5-8a708e5ec723)

Not sure why it was done like this 🤔 I feel all of them should be enabled by default, otherwise new users might not realize they need to enable them to get their balances